gensim/models/keyedvectors.py

```diff
@@ -1348,6 +1348,12 @@ def init_sims(self, replace=False):
         if getattr(self, 'vectors_norm', None) is None or replace:
             logger.info("precomputing L2-norms of word weight vectors")
             self.vectors_norm = _l2_norm(self.vectors, replace=replace)
+        elif (len(self.vectors_norm) == len(self.vectors)):  # if all of the vectors are precomputed
```

Comment:
Get rid of unnecessary parentheses.

```suggestion
        elif len(self.vectors_norm) == len(self.vectors):  # if all of the vectors are precomputed
```